rcl_lifecycle/test/test_default_state_machine.cpp

```diff
@@ -116,7 +117,7 @@ TEST_F(TestDefaultStateMachine, default_init) {
   // Because this init method is so complex, the succession of failures caused by a null
   // allocator will result in several error messages overwriting themselves.
   auto ret = rcl_lifecycle_init_default_state_machine(&state_machine, nullptr);
-  EXPECT_EQ(RCL_RET_ERROR, ret);
+  EXPECT_EQ(RCL_RET_ERROR, ret) << rcl_get_error_string().str;
```

Comment:
rcl_get_error_string().str is only output while ret != RCL_RET_ERROR.  
That is, no error string exists.  
So whether << rcl_get_error_string().str is needed ?